File: snippets/proxy/add-phone-number/add-phone-number.cs
using System;
using Twilio;
using Twilio.Rest.Proxy.V1.Service;

const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
const string authToken = "your_auth_token";

TwilioClient.Init(accountSid, authToken);

var phoneNumber = PhoneNumberResource.Create(
    sid: "PNXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
    pathServiceSid: "KSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
);

Console.WriteLine(phoneNumber.Sid);

File: snippets/proxy/add-phone-number/add-phone-number.js
// If you do not already have it installed, you will need twilio-node 
// installed to run this code: https://www.npmjs.com/package/twilio

const accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
const authToken = 'your_auth_token';
const client = require('twilio')(accountSid, authToken);

client.proxy.services('KSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
            .phoneNumbers
            .create({sid: 'PNXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'})
            .then(phone_number => console.log(phone_number.sid))
            .done();
